charts/chronicle/templates/_chronicle.tpl

```diff
@@ -52,23 +52,70 @@ chronicle: {{ include "common.names.fullname" . }}
 {{- end -}}
 
 {{- define "chronicle.id-provider.service" -}}
+{{- if .Values.IdProvider.service -}}
```

Comment:
I dont't see the point of this value.  The only place you seem to be using this template is when the devIdProvider.enabled==true( line 80 and line 97 ), in which case  you would not use some use supplied value you would use the actual service name of the devIdProvider.  So the conditionals here don't make any point either.